glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -239,16 +239,6 @@ definitions:
         description: |
           **Required.**
 
-          May be one of the following values:
-            - `<build date>`: An ISO date `yyyy-mm-dd` in UTC on which the
-              metric expires. For example, `2019-03-13`. This date is checked at
-              build time. Except in special cases, this form should be used so
-              that the metric automatically "sunsets" after a period of time.
-            - `never`: This metric never expires.
-            - `expired`: This metric is manually expired.
-        type: string
```

Comment:
I think we should still enforce `type: string` if we can.  Accepting anything could let through some hard-to-diagnose YAML silliness where it thinks things aren't strings.  If build id really will be a number, we could do:

```
anyOf:
    - type: string
    - type: number
```

(but I would argue build ids aren't numbers, even if they are composed of only numeric characters, and systems that assume they are always numbers lead to [this kind of thing](https://github.com/mozilla-mobile/fenix/blob/e6ee13dc/buildSrc/src/main/java/Config.kt#L55-L112).

Reply:
Fair enough. Strings it is.